packages/massmapper-app/src/MassMapperApp.tsx

```diff
@@ -103,12 +101,10 @@ const MassMapperApp: FunctionComponent<MassMapperAppProps> = observer(() => {
 					<Grid className={classes.mapContainer} item>
 						<ToolsOverlayComponent />
 						<MapContainer
-							center={center}
-							zoom={zoom}
 							className={classes.map}
 							scrollWheelZoom={true}
 							whenCreated={(map: Map) => {
-								mapService.initLeafletMap(map);
+								mapService.initLeafletMap(map, b);
```

Comment:
@sfarber not sure if this is fugly or ok . . . couldn't figure out how to elbow my own `extent` attr into something at the level of `center` or `zoom`.